.travis.yml

```diff
@@ -1,19 +1,43 @@
-sudo: false
 language: python
 python:
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
+  - 3.6
 env:
-  - DJANGO="1.11,<2.0"
-  - DJANGO="2.0,<3.0"
+  - DJANGO=1.7
+  - DJANGO=1.8
+  - DJANGO=1.9
+  - DJANGO=1.10
+  - DJANGO=1.11
+  - DJANGO=2.0
```

Comment:
This hard-codes Django 2.0 while the above will install 2.0.1 atm and 2.0.2 when it gets released next week.

Reply:
No, it doesn't. This is just a label.